### ISSUE TITLE
webpack implementation

### ISSUE THREAD

**Issue by cescoferraro** [lathonez/clicker#130]
Its [official](http://blog.ionic.io/announcing-ionic-2-beta-11/), Ionic is shifting to Webpack. This repo will become obsolete over time if it keeps using browserify.

I have been [trying](https://forum.ionicframework.com/t/setting-up-karma-on-ionic2-using-webpack/60438) but keep getting errors with @angular/testing

**Comment by lathonez** [comment]
Thanks for raising. I am aware of this, it was also raised [here](https://github.com/lathonez/clicker/issues/38#issuecomment-239497285) earlier.

What's requirement for implementing Webpack before beta.12 has been released? I may be able to help you with Karma (will reply on Ionic forum), but am not interested in making any changes here before Ionic do.

As a bit of history, this repo has been around since the very early days of Ionic 2 and we've always followed their build configuration as closely as possible:

* `gulp` (calling tsc directly)
* `webpack` (via gulp)
* `ionic-app-lib` (webpack wrapper, via gulp)
* `browserify`

If the new Webpack build is indeed released in `beta.12`, we'll be able to get a look at their implementation and mirror it as closely as possible for the test build.

If we try to implement anything before that there's a good chance we'll end up reworking the change to bring us closer to Ionic.

TL;dr - as soon as Ionic release a new build process, I'll move this over to mirror it as closely as possible, not before then.
